modules/tenant-management/src/Volo.Abp.TenantManagement.Application/Volo/Abp/TenantManagement/TenantAppService.cs

```diff
@@ -132,6 +136,10 @@ public virtual async Task<string> GetDefaultConnectionStringAsync(Guid id)
     public virtual async Task UpdateDefaultConnectionStringAsync(Guid id, string defaultConnectionString)
     {
         var tenant = await TenantRepository.GetAsync(id);
+        if (tenant.FindDefaultConnectionString() != defaultConnectionString)
+        {
+            await LocalEventBus.PublishAsync(new TenantChangedEvent(tenant.Id, tenant.NormalizedName));
+        }
```

Comment:
Should we publish the same event on the `DeleteDefaultConnectionStringAsync` method?